ownCloud/Client/Sharing/GroupSharingTableViewController.swift

```diff
@@ -347,7 +347,7 @@ class GroupSharingTableViewController: SharingTableViewController, UISearchResul
 
 	func updateSearchResults(for searchController: UISearchController) {
 		guard let text = searchController.searchBar.text else { return }
-		if text.count > core?.connection.capabilities?.sharingSearchMinLength?.intValue ?? 1 {
+		if text.count >= core?.connection.capabilities?.sharingSearchMinLength?.intValue ?? 2 {
```

Comment:
@hosy I think constant (e.g. defaultSharingSearchMinLength) would make sense instead of plain value 2